core/app/models/spree/wallet_payment_source.rb

```diff
@@ -1,19 +1,31 @@
 # frozen_string_literal: true
 
-class Spree::WalletPaymentSource < ActiveRecord::Base
-  belongs_to :user, class_name: Spree::UserClassHandle.new, foreign_key: 'user_id', inverse_of: :wallet_payment_sources
-  belongs_to :payment_source, polymorphic: true, inverse_of: :wallet_payment_sources
+module Spree
```

Comment:
can you please change the namespace/inheritance stuff into one or more other commits, so they are independently  documented? Also, why are these change needed?

Reply:
It's to follow the format used in the other model files.